draft-ietf-quic-load-balancers.md

```diff
@@ -875,10 +875,12 @@ handle these DCIDs, rests on some assumptions:
 * Incoming short headers do not contain DCIDs that are client-generated.
 * The use of client-generated incoming DCIDs does not persist beyond a few round
 trips in the connection.
-* In this interval, some exposed fields (e.g. UDP address and port, source and
-client-generated destination Connection ID) remain constant for all packets in
-the connection. For example, 0RTT and Initial packets use the same CIDs and
-ports.
+* While the client is using DCIDs it generated, some exposed fields (IP address,
+UDP port, client-generated destination Connection ID) remain constant for all
+packets sent on the same connection. While this document does not update the
+commitments in {{QUIC-INVARIANTS}}, the authors believe that the listed examples
+are a likely set of constants that load balancers can use with minimal risk of
+version-dependence.
```

Comment:
As this aims to represent IETF consensus, we should be able to say something more concrete about this than "the authors believe" implies.

I think that what can be said is that these assumption are made by this document and while it is not guaranteed that *every* QUIC version will conform to these assumptions, they are minimal and narrowly scoped.  This means that it is likely - and indeed desirable - for new versions of QUIC to conform to these expectations.  By doing so, a new version of QUIC will be able to use load balancers that conform to the design in this document.

That's a fairly compelling trade: make a small additional allowance to get all this stuff.  But it also means that you aren't constraining the design of QUIC versions that might not need or want involvement by a load balancer.

Reply:
OK, have another look.